src/components/ExitMovie/ExitMovie.js

```diff
@@ -0,0 +1,8 @@
+import './ExitMovie.css';
+import exitBanner from '../../images/old-school-return-banner.png'
+
+const ExitMovie = ({viewAll}) => {
+  return <img className='main-title' src={exitBanner} alt='old school theater banner with the title rancid tomatillos' onClick={() => viewAll()}/>
```

Comment:
do we need the closure here since we aren't passing anything in? We could just say `onClick={viewAll}`

Reply:
Not sure, I'll test it out and remove it if it's not required